pkg/csi/cinder/nodeserver.go

```diff
@@ -194,11 +194,10 @@ func getNodeIDMetdataService() (string, error) {
 func getNodeID() (string, error) {
 	// First try to get instance id from mount provider
 	nodeID, err := getNodeIDMountProvider()
-	if err == nil || nodeID != "" {
+	if err == nil {
 		return nodeID, nil
 	}
 
-	glog.V(3).Infof("Failed to GetInstanceID from mount data: %v", err)
```

Comment:
Ok, with removing additional logic, but why removing log??

Reply:
@adisky It's a redundant log based on the implementation of getNodeIDMountProvider https://github.com/kubernetes/cloud-provider-openstack/blob/20b7744ee505a1da05550a866051bf5323e4feba/pkg/csi/cinder/nodeserver.go#L235

Reply:
@hogepodg Thanks